packages/@expo/cli/src/start/server/middleware/ExpoGoManifestHandlerMiddleware.ts

```diff
@@ -207,7 +207,7 @@ export class ExpoGoManifestHandlerMiddleware extends ManifestMiddleware<ExpoGoMa
 
     Log.warn(
       APISettings.isOffline
-        ? 'Using anonymous scope key in manifest for offline mode with no cached development code signing info.'
+        ? 'Using anonymous scope key in manifest for offline mode.'
```

Comment:
Is the prior error no longer accurate?

Reply:
This will probably be shown more frequently to users, so having a less technical message is useful.